src/nuget.config

```diff
@@ -1,5 +0,0 @@
-<configuration>
-  <config>
-    <add key="repositoryPath" value="../lib" />
```

Comment:
I will have a private ceremony to say goodbye to this beautiful piece of configuration.